setup.cfg

```diff
@@ -11,7 +11,7 @@ license = Apache
 license_file = LICENSE
 keywords = physics fitting numpy scipy tensorflow pytorch jax
 project_urls =
-    Documentation = https://scikit-hep.org/pyhf/
+    Documentation = https://pyhf.readthedocs.io/
```

Comment:
Thoughts on changing it here too? Might not be a bad idea to direct people from PyPI to stable docs.

Reply:
this probably makes the most sense.